### ISSUE TITLE
Prompt is printed strangely on Windows

### ISSUE THREAD

**Issue by alexrp** [mono/sdb#22]
Something in `Mono.Terminal` isn't quite right on Windows. All prompts seem to have a bunch of white space appended to them.

**Comment by alexrp** [comment]
Only happens when colors are enabled. Seems to happen on all OSs when `Mono.Terminal` is used.

**Comment by alexrp** [comment]
Example:

    Welcome to the Mono soft debugger (sdb 1.4.5485.30864)   
    Type 'help' for a list of commands or 'quit' to exit     

    (sdb)          cfg list
    ...

**Comment by elbeardmorez** [comment]
on my GNU/Linux box, I had this behaviour before installing libedit library. incidentally, i had to manually link (ln -s /path/to/libedit.so.0.0.36 /path/to/sdb/libedit.so.2) the library as I couldn't find the ABI version searched for (.so.2). note also that the fairly standard path /usr/local/lib64 wasn't searched (according to strace) so perhaps a build dependency is in need :)

**Comment by saper** [comment]
Same here with the FreeBSD install. I have edited `bin/sdb.exe.config` to point to `libedit.so` and it was fine after that.

**Comment by alexrp** [comment]
Yes, this issue is specifically about the managed `Mono.Terminal.LineEditor` class.
